examples/configs/tasks/DSC_obs.yml

```diff
@@ -0,0 +1,43 @@
+task_id: DSC_obs
+feature_of_interest: RT
+stimulus_id: DSC_task_1
+instruction_id: dsc_1
+device_id_array:
+- Intel_D455_1
+- Intel_D455_2
+- Intel_D455_3
+- Mbient_LH_2
+- Mbient_RH_2
+- Mbient_RF_2
+- Mbient_LF_2
+- Mbient_BK_1
+- FLIR_blackfly_1
+- Mic_Yeti_dev_1
+- Eyelink_1
+- IPhone_dev_1
+sensor_id_array:
+- - Intel_D455_rgb_1
+  - Intel_D455_depth_1
+- - Intel_D455_rgb_1
+  - Intel_D455_depth_1
+- - Intel_D455_rgb_1
+  - Intel_D455_depth_1
+- - mbient_acc_1
+  - mbient_gyro_1
+- - mbient_acc_1
+  - mbient_gyro_1
+- - mbient_acc_1
+  - mbient_gyro_1
+- - mbient_acc_1
+  - mbient_gyro_1
+- - mbient_acc_1
+  - mbient_gyro_1
+- - FLIR_rgb_1
+  - ''
```

Comment:
Does the sensor nesting actually help in some way? Would like to avoid the empty '' if possible and wondering if we need the structure beyond a flat list.

Alternatively, do we even need to specify the sensors? If we specify the devices then we should know the corresponding sensor set. What happens if we have a mismatch between what is specified here and what the devices require?

Reply:
Again, this is from an automated pull from the database.  The structure you're referring to was defined in the db. I think we can remove this, but, in general and in light of the complete lack of automated tests, there are far too many changes involved in the simplest transfer of data out of the db and into the yaml files to pile on a bunch of refactorings. IMO.

Reply:
I had not considered that this was an automated DB export (which makes sense). I would agree with making this a story for a future refactoring that is reliant on the introduction of more tests.